book/website/community-handbook/content.md

```diff
@@ -0,0 +1,139 @@
+Structurally, all Guides in _The Turing Way_ Github repository are folders and live within the `book/website` directory. 
+Depending on the type of content you wish to contribute, the files you create should be added as recommended below:
+
+### Case Studies
+
+For every Guide, there is a `Case Studies` chapter that contains a collection of case studies that explore how key concepts from the Guide relate to specific subjects (such as a project, person, organisation, or phenomenon).
+
+Case study files are located in the `case-studies` sub-folder within each Guide.
+
+For example, the case study about the [Statistical Methods Manuscript](https://the-turing-way.netlify.app/reproducible-research/case-studies/statistical-methods-manuscript.html) in the Guide for Reproducible Research follows this file structure:
+
+```
+book/website
+│
+└───reproducible-research <---- (Folder for the Guide to Reproducible Research)
+│   │
+│   └───binderhub
+│   │
+│   └───case-studies <---- (folder containing subchapter for case studies the Guide to Reproducible Research)
+│   |   │   statistical-methods-manuscript.md <---- (Statistical Manuscripts Case Study)
+|   |   |   example-case-study.md
+|   |   |   ...
+|   |
+|   |   ...
+|   |   case-studies.md <---- (Landing page for Case Studies chapter)
+|   |   ...
+│   
+└───project-design <---- (Folder for the Guide for Project Design)
+    │   project-design.md
+    │   ...
+```
+
+More case studies (such as `example-case-study-file.md`) for the Guide for Reproducible Research can be added in the same location as `statistical-methods-manuscript.md`.
+
+### Chapters
+
+Chapters are sub-folders within Guides.
+
+For example, the [Version Control](https://the-turing-way.netlify.app/reproducible-research/vcs.html) chapter in the Guide for Reproducible Research has the following structure:
+
+```
+book\website
+│
+└───reproducible-research <---- (folder for the Guide to Reproducible Research)
+│   │   reproducible-research.md ---- (Guide's Landing Page)
+│   │   vcs.md <---- (Landing page for the Version Control chapter)
+│   │   example-chapter.md (landing page for a new chapter)
+|   |
+│   └───vcs (chapter folder)
+│   |   │   vcs-workflow.md
+|   |   |   vcs-git.md
+|   |   |   vcs-git-commit.md
+|   |   |   ...
+|   |   |   vcs-personal-stories.md
+│   |   │   vcs-checklist.md
+│   |   │   vcs-resources.md
+|   └───example-chapter (new chapter folder)
+│       │   example-subchapter.md
+|       |   ...
+|       |   example-personal-stories.md
+│       │   example-checklist.md
+│       │   example-resources.md
+│   
+└───project-design <---- (folder for the Guide for Project Design) 
+    │   project-design.md
+    │   ...
+```
+
+To add more chapters to the Guide for Reproducible Research, 
+use the file locations of `example-chapter`'s constituent files for reference.
+
+(ch-content-checklist)=
+## Contribution Checklist
+
+### Follow the style and consistency guidelines
```

Comment:
Since this might become a workflow, we can specify it further. So the very first checkpoint would be to pick up a chapter or case study template _if the person is starting an entirely new one_.